Added toluene shock tube yaml files

morganmayer opened the pull request.

Commit:
Added toluene shock tube yaml files

Commit:
Validated all toluene files with chemked version 0.3.0 and changed chemked-version line in each file

bryanwweber commented on Toluene/Burcat 1986 yaml files/Burcat_mixA.yaml:
```diff
@@ -0,0 +1,149 @@
+file-version: 0
+chemked-version: 0.3.0
+reference:
+    doi:
```
I think we should remove fields that are blank from the files. IIRC, this should actually raise a validation error...

bryanwweber commented on Toluene/Burcat 1986 yaml files/Burcat_mixB.yaml:
```diff
@@ -0,0 +1,177 @@
+file-version: 0
+chemked-version: 0.3.0
+reference:
+    doi: 19860015959
```
I don't think this is a valid DOI?

morganmayer replied:
This Burcat paper did not have a doi since it was a technical memo but I did validate it with another valid doi from another paper, then I believe @kyleniemeyer used some get-around for using the files with pyteck. Should I take out the doi line for the Burcat files then?

bryanwweber replied:
Yeah, I think that would be best until we decide what to do about papers like this without a permanent identifier.

kyleniemeyer replied:
We had discussed using a URL for cases without a DOI; in this case, it would be https://ntrs.nasa.gov/search.jsp?R=19860015959 (so we know what that is moving forward)

bryanwweber commented on Toluene/Shen 2009 yaml files/Shen_toluene_phi1.0_50atm.yaml:
```diff
@@ -0,0 +1,141 @@
+        type: d/dt max
+datapoints:
+    - temperature:
+        - 1021.0 kelvin
```
The authors give estimates of the uncertainties of these quantities in their paper, so we should include them here. See the bottom right of pg. 166 and pg. 167

bryanwweber commented on Toluene/Vasudevan 2005 yaml files/Vasudevan_phi_0.5.yaml:
```diff
@@ -0,0 +1,87 @@
+        target: OH
+        type: 1/2 max
+datapoints:
+    - temperature:
```
Same here, uncertainty estimates are given on pg. 1156

morganmayer replied:
I haven't used uncertainties in a file before.  Is this a line that goes under each data point or in the common properties?

kyleniemeyer replied:
Check out the bit in here: https://pr-omethe-us.github.io/PyKED/ck-tutorial.html#the-data-section

bryanwweber commented on Toluene/Davidson 2005 yaml files/Davidson_2005_toluene_phi0.5_50atm.yaml:
```diff
@@ -0,0 +1,78 @@
+        target: OH*
+        type: d/dt max
+datapoints:
+    - temperature:
```
Uncertainties on pp. 1176 and 1177

morganmayer commented:
@kyleniemeyer I've made some edits to some ignition delay definitions after realizing many should be d/dt max extrapolated.  git status says the file methyl-pentanoate/phi=2.0/15-bar/Tc_846K_P0_0.5419_T0_413K_chemked.yaml was modified, but I don't know why. I used git diff to see the changes and it looked like the whole file was new.  Should I remove this file before committing my changes or commit and push it anyways?

kyleniemeyer commented:
@morganmayer you should be able to just not add that methyl pentanoate file to your commit—meaning, only include the files in the commit that you actually changed.

kyleniemeyer commented:
If you need to remove it from staging, try `git rm --cached filename`

kyleniemeyer commented:
Sorry, should probably be `git reset HEAD -- filename`

Commit:
Changed file names and added Mittal and Sung RCM files

morganmayer commented:
I fixed some file names and double checked ign delay definitions, and added the uncertainties. I just need to figure out some data errors in the Mittal and Sung RCM files, then this set should be complete.

Commit:
deleted Mittal RCM files. Will add in a separate branch

Commit:
changed file-author to file-authors

Commit:
validated all Toluene files with pyked 0.4.1

morganmayer commented:
@bryanwweber I apologize for the delay. I made the format changes and validated the toluene files with pyked 0.4.1. This pull request should be ready to submit unless there are any other changes you can think of.

bryanwweber committed:
NTRS URLs are not valid DOIs

Move the NTRS URL for Burcat toluene to the Detail field

bryanwweber committed:
Standardize YAML format

bryanwweber commented:
Hi @morganmayer Sorry for the long delay! @kyleniemeyer just reminded me these were still open. In this PR, it looks like the mole fractions of the Burcat data don't sum to 1.0 (this is the case for all the Burcat files, although mixes A, B, and D will validate). Can you check this and make a reasonable determination about how to resolve the issue?

Commit:
fixed mole fractions

morganmayer commented:
Hi @bryanwweber, I fixed the Burcat mole fractions so they all sum to 1 now. I used the exact toluene and O2 mole fractions reported and made the remainder argon. I just validated them with ChemKED version 0.4.1, so they should be correct now.

bryanwweber commented:
Thanks @morganmayer! Nice work here!
